caddy/content.md

```diff
@@ -20,6 +20,14 @@ In the examples below, a [named volume](https://docs.docker.com/storage/volumes/
 
 Note that named volumes are persisted across container restarts and terminations, so if you move to a new image version, the same data and config directories can be re-used.
 
+### Linux capabilities
+
+Caddy ships with HTTP/3 support enabled by default. To improve the performance of this UDP based protocol, the underlying quic-go library tries to increase the buffer sizes for its socket. The `NET_ADMIN` capability allows it to override the low default limits of the operating system without having to change kernel parameters via sysctl.
```

Comment:
:man_shrugging: you'd certainly know better than I, Docker+kernel+sys interactions are not an area I'm knowledgeable with.

But I think `NET_ADMIN` would allow the software to automatically fix the limits automatically if it finds it needs to, rather than trying to give a one-size-fits-all number. But this isn't our code doing it, it's code from quic-go, a dependency which provides QUIC and HTTP/3 support.

Reply:
I'll keep this as a draft and test with the final 2.7.